### ISSUE TITLE
Unable to locate losses in the code 

### ISSUE THREAD

**Issue by msohaildanish** [AmingWu/Single-DGOD#6]
Hi Aming Wu,

Thank you for sharing the code. I am trying to reproduce the results mentioned in your paper but currently I am unable to locate your method in the code e.g the self-distillation loss. Is the shared code only contains Faster RCNN implantation?

**Comment by Yaser-wyx** [comment]
Yes, I have the same issue I can't locate your method in the code, and the code only contains Faster RCNN.
